pandas/core/window/rolling.py

```diff
@@ -602,6 +606,25 @@ def calc(x):
                 )
                 self._check_window_bounds(start, end, len(x))
 
+                arithmetic_operators = [
```

Comment:
So this may still be addressable in cython without hardcoding this here which is also not idea.

Going back to a couple of commits earlier where `signbit` was the issue, all it does is check if the value is negative: https://www.cplusplus.com/reference/cmath/signbit/

So maybe test if you can do a value comparison to make it complex compatible